controller/controlplane/controller.go

```diff
@@ -395,6 +395,7 @@ func (r *Reconciler) constructControlPlaneManagerConfigOptions(
 		WithAnonymousReports(r.AnonymousReportsEnabled),
 		WithAnonymousReportsFixedPayloadCustomizer(payloadCustomizer),
 		WithClusterDomain(r.ClusterDomain),
+		WithReverseSync(lo.FromPtr(lo.FromPtr(cp.Spec.DataPlaneSync).EnableReverse)),
```

Comment:
I feel that adding a `if cp.Spec.DataPlaneSync` here would be slightly more readable.

That `if` branch would contain other dp sync related options so it would make it visually grouped.

WDYT?

Reply:
you're right PTAL https://github.com/Kong/kong-operator/pull/1891/commits/befe70d74766b9c89d8a5dba8a915fdef5672ca9